test/grape_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+require 'grape'
+require 'grape/active_model_serializers'
+
+class ActiveModelSerializers::GrapeTest < Minitest::Test
+  include Rack::Test::Methods
+
+  class GrapeTest < Grape::API
+    format :json
+    formatter :json, Grape::Formatters::ActiveModelSerializers
+    helpers Grape::Helpers::ActiveModelSerializers
+
+    resources :grape do
+      get '/render' do
+        render ARModels::Post.new(title: 'Dummy Title', body: 'Lorem Ipsum')
+      end
+
+      get '/render_with_json_api' do
+        post = ARModels::Post.new(title: 'Dummy Title', body: 'Lorem Ipsum')
+        render(
+          post,
+          meta: { page: 1, total_pages: 2 },
+          adapter: :json_api
+        )
+      end
+    end
+  end
+
+  def app
+    GrapeTest.new
+  end
+
+  def test_formatter_returns_json
+    get '/grape/render'
+    assert last_response.ok?
+    assert JSON.parse(last_response.body)
+  end
+
+  def test_render_helper_passes_through_options_correctly
+    get '/grape/render_with_json_api'
+
+    post = ARModels::Post.new(title: 'Dummy Title', body: 'Lorem Ipsum')
+    serializer = ARModels::PostSerializer.new(post)
+    adapter = ActiveModel::Serializer::Adapter::JsonApi.new(serializer, meta: { page: 1, total_pages: 2 })
```

Comment:
This should be

```ruby
serializable_resource = ActiveModel::SerializableResource.new(post, serializer: ARModels::PostSerializer, adapter: :json_api, meta: { page: 1, total_pages: 2})`
assert last_response.ok?
assert_equal serializable_resource.to_json, last_response.body
```

or, using the `serializable` test helper which is the same, but less verbose https://github.com/rails-api/active_model_serializers/blob/master/test/support/serialization_testing.rb#L34-L35

and follow the same pattern in https://github.com/rails-api/active_model_serializers/pull/1336/files#r45619253